exercises/bowling/canonical-data.json

```diff
@@ -1,117 +1,110 @@
 {
   "#": [
-    "Bowling is game where players roll a heavy ball to knock down pins",
-    "arranged in a triangle. Write code to keep track of the score of a",
-    "game of bowling."
+    "Students should implement roll and score methods.",
+    "Roll should accept a single integer.",
+    "Score should return the game's final score, when possible",
+    "For brevity the tests all the rolls in an array;",
```

Comment:
This sentence might be missing a word, and I can't figure out what it should be. Holds? Keeps? Stores?